server/controllers/api/config.ts

```diff
@@ -10,6 +10,7 @@ import { customConfigUpdateValidator } from '../../middlewares/validators/config
 import { ClientHtml } from '../../lib/client-html'
 import { auditLoggerFactory, CustomConfigAuditView, getAuditIdFromRes } from '../../helpers/audit-logger'
 import { remove, writeJSON } from 'fs-extra'
+import { version } from '../../tools/cli'
```

Comment:
@Chocobozzz argh that should have been `import { getVersion } from '../../helpers/utils'` (and then modify the call below)

Reply:
https://github.com/Chocobozzz/PeerTube/commit/913f08f2bd5b03c121ee65751e938e4faa27e7f1